audioswitch/src/main/java/com/twilio/audioswitch/bluetooth/BluetoothHeadsetReceiver.kt

```diff
@@ -55,7 +57,19 @@ internal class BluetoothHeadsetReceiver(
                                 "Bluetooth ACL device " +
                                         bluetoothDevice.name +
                                         " disconnected")
-                        headsetCache.remove(BluetoothHeadset(bluetoothDevice.name))
+
+                        headsetState.state = when {
+                            headsetManager.hasActiveDevice() -> {
+                                HeadsetState.State.Activated
```

Comment:
just for my understanding, these block covers the case when a user has an active BT device, maybe disconnects another non-active BT device?